website/api-gateway/package.json

```diff
@@ -21,7 +21,7 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@gear-js/interfaces": "1.0.0",
+    "@gear-js/interfaces": "workspace:^",
```

Comment:
I guess it's better to leave this dependency in the packages' dependencies. Because they may use different versions at different times.

Reply:
Point taken; versions changed back.